api/src/main/java/net/kyori/adventure/title/Title.java

```diff
@@ -164,5 +199,109 @@
      * @since 4.0.0
      */
     @NotNull Duration fadeOut();
+
+    /**
+     * Creates a new builder initialized with the attributes of this times.
+     *
+     * @return the builder
+     * @since 4.10.0
+     */
+    @Override
+    default @NotNull Builder toBuilder() {
+      return new TitleImpl.TimesImpl.BuilderImpl()
+        .fadeIn(this.fadeIn())
+        .stay(this.stay())
+        .fadeOut(this.fadeOut());
+    }
+
+    /**
+     * A builder for a {@link Title.Times}.
+     *
+     * @since 4.10.0
+     */
+    interface Builder extends Buildable.Builder<Title.Times> {
+      /**
+       * Set the time the title will fade-in.
+       *
+       * @param fadeIn the fade-in time
+       * @return this
+       * @since 4.10.0
+       */
+      @Contract("_ -> this")
+      @NotNull Builder fadeIn(final @NotNull Duration fadeIn);
```

Comment:
Maybe add in `fadeInSeconds` overloads here?

@kashike thoughts on adding tick-based convenience overloads here? or do we not want to depend on a fixed tick rate?

Reply:
Would prefer no tick-based methods.

Reply:
> fadeInSeconds
Sounds like something the Duration class handles fairly comprehensively to me 🤷‍♂️